data/json/items/ranged/archery.json

```diff
@@ -357,6 +357,32 @@
     "durability": 4,
     "clip_size": 1
   },
+  {
+    "id": "longbow_makeshift",
+    "type": "GUN",
+    "symbol": "(",
+    "color": "yellow",
+    "name": { "str": "makeshift longbow" },
+    "description": "A terribly bulky and awkward effort at recreating an effective English weapon. Much larger, harder to draw, and more accurate than your previous attempts, but you still feel embarrassed just looking at it.",
```

Comment:
Player might not done any "previous attempts".

Reply:
Good catch. I'll change the description, thanks.

Reply:
Yeah, that's good.